scipy/optimize/tests/test_bracket.py

```diff
@@ -155,20 +155,29 @@ def f(xs, js):
             funcs = [lambda x: x - 1.5,
                      lambda x: x - 1000,
                      lambda x: x - 1000,
-                     lambda x: np.nan]
+                     lambda x: np.nan,
+                     lambda x: x,
+                     lambda x: x,
+                     lambda x: x]
```

Comment:
I think we only need to have one example of each status code here; I'd prefer that rather than have one example of the first few and many examples of the last one - just as a matter of consistency. My intent of the test was to show that the different codes could all be different; it really is elementwise. There could be a separate test for exploring many ways of getting a given status code.